### ISSUE TITLE
Accuracy lower than expected

### ISSUE THREAD

**Issue by iamGavinZhou** [titu1994/DenseNet#5]
If i use the model you provided(DenseNet-40-12-Cifar10), the classification accuracy is 94.74%.
But i try to train a  new model, i just uncomment "model.fit_generator(...)" line in Cifar10.py, after 200 epoches, accuracy is 88.3%(seen from the screen).
Is there something wrong or how to train a new model from original Cifar10 dataset?
Thanks!

**Comment by titu1994** [comment]
As I have mentioned in an older issue (which you can also see in the closed issues section), I split training into 3 stages.

1) Stage 1: Train for 100 epochs at Adam Learning rate at 1e-3
2) Stage 2: Train for 100 epochs at Adam Learning rate at 1e-4
3) Stage 3: Train for 50 epochs at Adam Learning rate at 1e-5.

Each time, the accuracy goes up a little. Stage 1 -> 2 is approximately 2-3 % jump. Stage 2 -> 3 is 1-1.5 % jump
